templates/documentation/taspr.html

```diff
@@ -154,6 +154,27 @@ <h4>Yearly values (point query)</h4>
 
 <h4>Yearly values within provided year range (point query)</h4>
```

Comment:
```suggestion
<h4>Monthly min, mean, and maximum temperature for each year (point query)</h4>
```
Update this `<h4>` heading to match the corresponding output section.